CreateImage soeed uo x10; turn off image cache

PiotrZub opened the pull request:
After create special non stream implementation of
MemoryRandomAccessStream images is created ten times faster.
That is why the cache is not necesary and I think it wasn't working
because of problem with hashCode.

FabricioCabeca commented:
The image creation was slow indeed, if this fixes the issue there shouldn't be a need for a cache. Thanks !

PiotrZub committed:
CreateImage soeed uo x10; turn off image cache

After create special non stream implementation of
MemoryRandomAccessStream images is created ten times faster.
That is why the cache is not necesary and I think it wasn't working
because of problem with hashCode.
